lte/gateway/deploy/roles/ovs_deploy/vars/all.yaml

```diff
@@ -15,14 +15,20 @@ ovs_build_config:
   # ovs version on github default v2.8.1
   ovs_version: "v2.8.10"
   # Patches you want to apply
-  packages:
+  packages_debian:
     - "oai-gtp_4.9-5_amd64.deb"
     - "libopenvswitch_2.8.10-1_amd64.deb"
     - "openvswitch-datapath-dkms_2.8.10-1_all.deb"
     - "openvswitch-datapath-source_2.8.10-1_all.deb"
     - "openvswitch-common_2.8.10-1_amd64.deb"
     - "openvswitch-switch_2.8.10-1_amd64.deb"
 
+  packages_ubuntu:
+    - "libopenvswitch_2.14.3-1_amd64.deb"
+    - "openvswitch-datapath-dkms_2.14.3-1_all.deb"
+    - "openvswitch-common_2.14.3-1_amd64.deb"
+    - "openvswitch-switch_2.14.3-1_amd64.deb"
+
```

Comment:
I do not see OVS packages getting installed in magma_focal VM with this patch applied.